style.css

```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins');
+
+@media screen and (min-width: 480px) {
+  body {
+    background-color: lightgreen;
+  }
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  overflow: hidden;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.flexbox {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px;
+  background-color: #fff;
+}
+
+.logo {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  padding: 10px 12px;
+  text-decoration: none;
+}
+
+.fa-bars {
+  color: #6070ff;
+  width: 18px;
+  height: 12px;
+}
+
+.intro {
+  background-image: url('img/hbackground.png');
+  border-radius: 0 0 0 50px;
+  background-color: #fff;
+  padding: 20px;
+}
+
+.description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.action-text {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  text-decoration: none;
+}
+
+.social-media li {
+  display: inline;
+}
+
+.social-media {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+.fa-brands {
+  color: #505f79;
+}
+
+.title {
+  font-style: normal;
```

Comment:
- [ ] As the previous reviewer mentioned please make sure you stick to the `Figma`  here you can use `font-style: bold`.
your design 👇
![portfigma](https://user-images.githubusercontent.com/66992901/191591517-0fb4cf5b-713f-4477-b10e-f0b5de93562d.PNG)

Figma design 👇
![figma1](https://user-images.githubusercontent.com/66992901/191591576-b139bf36-bbd4-476f-a555-ba00a1ac6fd9.PNG)